### PR TITLE
Fix double label column bug

### PR DESCRIPTION
**Pull request by tuethan1999**
### Pull Request Description
Fix double label column bug by removing duplicate columns before merging
-----
*After creating the pull request: in order to pass the **changelog_updated** check you will need to update the "Future Release" section of* `docs/source/changelog.rst` *to include this pull request.*

### CODE REVIEW

File: docs/source/changelog.rst [hunk 1]
```diff
@@ -9,6 +9,7 @@ Changelog
     * Fixes
         * Fix ``encode_features`` converting excluded feature columns to a numeric dtype (:pr:`1123`)
         * Improve performance of unused primitive check in dfs (:pr:`1140`)
+        * Fix duplicate label column generation when labels are passed in cutoff times and approximate is being used (:pr:`1147`)
```
Comment (tamargrey) [inline docs/source/changelog.rst:12]: This should get moved to the Future Release section once you rebase/merge in main

File: featuretools/computational_backends/calculate_feature_matrix.py [hunk 2]
```diff
@@ -437,6 +437,7 @@ def update_progress_callback(done):
                 # if approximate, merge feature matrix with group frame to get original
                 # cutoff times and passed columns
                 if approximate:
+                    pass_columns = [c for c in pass_columns if c not in _feature_matrix]
```
Comment (kmax12) [inline featuretools/computational_backends/calculate_feature_matrix.py:440]: don;t forget a test case

**Comment by rwedge** [comment]
In order to satisfy the CLA check, all the EC2 user commits will need to be updated to be authored by an actual github user.  Alternatively, you could create a new branch and PR with the same code but valid github email for the author

**Comment by rwedge** [comment]
Moved to #1160

**Pull request closed without merging**